src/com/pentalog/pentastagiu/homework2/BankAccount.java

```diff
@@ -0,0 +1,30 @@
+package com.pentalog.pentastagiu.homework2;
+
+public class BankAccount {
+    private int id;
+    private double balance;
+
+    public BankAccount() {
+        this.balance = 0;
+    }
+
+    public BankAccount(double balance) {
+        this.balance = balance;
+    }
+
+    public void withdraw(double amount) {
+        if (amount > balance) {
+            System.out.println("You don't have enough funds");
+        } else {
+            balance -= amount;
+        }
+    }
+
+    public void deposit(double amount) {
+        balance += amount;
```

Comment:
dar daca clientul iti trimite ca parametru un amount negativ? vei obtine un comportament neastptat, ceea ce inseamna ca si de aici lipseste o validare de amount negativ(la fel si in withdraw)